### DISCUSSION TITLE
Add support for Sia alerts

### DISCUSSION THREAD

**Discussion by tbenz9** [siacentral/host-manager#104] [Feature Requests]
/daemon/alerts is a new API endpoint that is meant to tell users what is wrong with any modules they are running.  It would be awesome to display those alerts somewhere.

https://gitlab.com/NebulousLabs/Sia/merge_requests/3769

**Comment by n8mgr** [comment]
The new alerts in Sia will definitely be displayed right alongside the current alerts in Host Manager once its more finalized and ready on the Sia side.

**Comment by n8mgr** [comment]
Removing the milestone, Sia's built in alerts only support one host alert at the moment and Host Manager already has a similar alert. At this point Host Manager's alerts are much more robust. Will look at again at a later date.

Sia's Alerts:
+ HostInsufficientCollateral

Host Manager Alerts:
+ ContractsNotAccepted
+ NotEnoughCollateral
+ IncorrectConsensus
+ FailedContracts
+ IncorrectContractStatus
+ ConnectionError
+ StorageFolderFailedReads
+ StorageFolderFailedWrites
+ StorageFull
+ StorageAlmostFull
+ StorageNearFull
+ StorageUnder4TB
+ NoStorageAvailable
+ WalletLocked
+ WalletRescanning
+ WalletEmpty
